src/components/common/SearchInput/SearchInput.tsx

```diff
@@ -40,26 +64,67 @@ const SearchInput = React.forwardRef<SearchInputRef, SearchInputProps>(
     { onSearchClick, buttonTestId = 'store-search-button', ...props },
     ref
   ) {
+    const [searchQuery, setSearchQuery] = useState<string>('')
```

Comment:
What do you think about removing this state and only using the suggestions.length value to render the Suggestion component on line 120.

Reply:
Yeah, it works, but I used the state because when the user clicks on any suggestion, the suggestions are to be the search query, so that should be the value of the input.

Reply:
Ok ok! I see, thank you for the answer.